app/settings.py

```diff
@@ -24,7 +24,8 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = ['localhost', 'hackupc.com', 'my.hackupc.com']
+
+ALLOWED_HOSTS = ['localhost', 'hackupc.com', 'my.hackupc.com', '127.0.0.1']
```

Comment:
I guess @alvarogirona wanted to use it as host to try production mode? Idk, didn't add it myself I always use localhost